tests/typechecker/return-value-type-mismatch.jakt

```diff
@@ -1,3 +1,3 @@
 function foo() -> u64 {
-        return "Foo"
+    return "Foo"
```

Comment:
Please fix these in the commit that added them instead if introducing weird formatting and fixing it in a later commit

Reply:
That makes sense, of course. Thanks! Fixed it, I think